src/components/global-view-core.js

```diff
@@ -52,6 +52,22 @@ const modalStyle = {
     p: 4,
 };
 
+const modalNoOverflowStyle = {
```

Comment:
modal style is used by both modals "events" and "files" ... I can't change one without having an impact on the other

Reply:
Yes but the overflow should not be on the entire modal in both cases because it is better to keep the title and the close button visible when you scroll